pointercrate-demonlist-pages/static/js/statsviewer/individual.js

```diff
@@ -143,6 +143,16 @@ class IndividualStatsViewer extends StatsViewer {
       })
     );
   }
+
+  selectPlayerByID(id) {
+    return get(this.retrievalEndpoint + parseInt(id) + "/").then((data) => {
+      if (data.data.data.banned) { // wtf
+        this.setError("This player is banned!")
+        return;
+      }
+      this.onReceive(data);
+    });
```

Comment:
I don't actually think we really need this check. For banned players, the profile will just be empty and that's that.